src/main/java/com/github/vkorobkov/jfixtures/util/StringUtil.java

```diff
@@ -1,15 +1,32 @@
 package com.github.vkorobkov.jfixtures.util;
 
+import lombok.val;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
 public final class StringUtil {
     private StringUtil() {
     }
 
-    public static String removePrefixes(String s, String ... prefixes) {
-        for (String prefix: prefixes) {
+    public static String removePrefixes(String s, String... prefixes) {
+        for (String prefix : prefixes) {
             if (s.startsWith(prefix)) {
                 return s.substring(prefix.length());
             }
         }
         return s;
     }
+
+    public static Path cutOffExtension(Path filePath) {
```

Comment:
Can we rename `filePath` to `path`, `fileName` to `name`, `parent` to `location`?